core/storage/learner_group/gae_models.py

```diff
@@ -205,7 +205,11 @@ def export_data(cls, user_id: str) -> Dict[str, LearnerGroupDataDict]:
                     'subtopic_page_ids': [],
                     'story_ids': []
                 }
-            elif user_id in learner_group_model.facilitator_user_ids:
+            # We add a no branch flag here because if found_models is not None,
```

Comment:
Put quotes around 'no branch'.

Explain more, in the comment, about why you came to this conclusion.

Ditto below.

Reply:
We fetch `found_models` using datastore_services filtering on the condition that **user_id** should be equal to either of the **learner_user_ids, invited_learner_user_ids or facilitator_user_ids**. If `found_models` is not None, it means that **user_id** has to be in atleast one of those and the else part of this if/elif/else chain will never occur. Hence, we use a `no branch` flag. Ditto below.

> Also, if this is a conclusion should the solution be to replace elif with else?
I normally avoid tampering with the code itself -- but replaced elif with else here,